resolwe/flow/static/flow/typeSchema.json

```diff
@@ -241,19 +241,19 @@
                 }
             }
         },
-        "list_data": {
+        "list_integer": {
             "type": "object",
             "required": ["type", "value"],
             "additionalProperties": false,
             "properties": {
                 "type": {
                     "type": "string",
-                    "pattern": "^list:data:[a-z0-9:]+:$"
+                    "pattern": "^list:(basic:integer|data:[a-z0-9:]+):$"
```

Comment:
Thanks for pointing this out. It is left-over from the old platform, where ids were uuids. I've fixed it and added test